### DISCUSSION TITLE
How does a route match all addresses under a certain path

### DISCUSSION THREAD

**Discussion by Fgaoxing** [falconry/falcon#2144] [Q&A]
How does a route match all addresses under a certain path.It's Not in the document

**Comment by CaselIT** [comment]
Hi,

Version 3.x does not have a builtin field converter for this.
In v4 there will be the path field converter https://falcon.readthedocs.io/en/latest/api/routing.html#falcon.routing.PathConverter

In the mean time you can use one of there recepies:
https://github.com/falconry/falcon/issues/648#issuecomment-546670652
or
https://falcon.readthedocs.io/en/latest/user/recipes/raw-url-path.html#raw-url-path-recipe

**Comment by Fgaoxing** [comment]
Thanks a lot!
